megazords/full/android/build.gradle

```diff
@@ -72,6 +72,8 @@ dependencies {
     }
 }
 
+ext.configureUniFFIBindgen("full_megazord")
```

Comment:
@bendk this results in

```
Build file '/builds/worker/checkouts/vcs/megazords/full/android/build.gradle' line: 75

* What went wrong:
A problem occurred evaluating project ':full-megazord'.
> No signature of method: org.gradle.internal.extensibility.DefaultExtraPropertiesExtension.configureUniFFIBindgen() is applicable for argument types: (String) values: [full_megazord]
```

any ideas?

Reply:
I checked this out and there was a lot of gradle work needed for this.  I opened #6610 for it.